tests/PySys/environments/environment_c8y.py

```diff
@@ -69,6 +70,35 @@ def get_thin_edge_device_by_name(self, device_id: str):
                 return device
         return None
 
+    def get_child_device_of_thin_edge_device_by_name(self, thin_edge_device_id: str, child_device_id: str):
+        self.device_fragment = self.get_thin_edge_device_by_name(
+            thin_edge_device_id)
+        internal_id = self.device_fragment['id']
+        child_devices = self.to_json_response(requests.get(
+            url="{}/inventory/managedObjects/{}/childDevices".format(self.c8y_url, internal_id), auth=self.auth))
+        for child_device in child_devices['references']:
+            if child_device_id in child_device['managedObject']['name']:
+                return child_device['managedObject']
+
+        return None
+
+    def get_last_measurements_from_device(self, device_internal_id: str):
```

Comment:
The name is a bit confusing since we only return one measurement. What about get_last_measurement_from_device?